### ISSUE TITLE
incorrect incomplete tests?

### ISSUE THREAD

**Issue by goatslacker** [coveraje/coveraje#12]
related to branches not being counted

here's some example code:

```
if (/^he/.test('hello')) {
  return true;
}
```

Will set the `if` as "incomplete" whereas

```
 if (/^he/.test('hello') === true) {
  return true;
}
```

will mark it as complete.

**Comment by goatslacker** [comment]
This doesn't happen with function calls or with object properties (even if those properties are functions)

Example

```
foo = true;
if (foo) {
}

foo = { bar: function () { return true } };
if (foo.bar()) {
}

foo = function () { return true };
if (foo()) {
}
```

all of them are marked as completed

**Comment by goatslacker** [comment]
It breaks with `isNaN` and `RegExp.test`

but it works with `!Number.NaN`

**Comment by WolfgangKluge** [comment]
It's the last ) inside the condition that breaks the code. I'll fix it soon. Thanks for report

**Comment by WolfgangKluge** [comment]
Sorry that it took so long.
Fixed in 5e4b1881dafd74ce33f4167e4c47c62052201a9a
